src/table/paginator.scss

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+@import '../button/index';
+@import './variables';
+
+@mixin iui-paginator {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  flex: 1;
+  justify-content: space-around;
+  align-items: center;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
+  }
+
+  > .iui-left,
+  > .iui-center,
+  > .iui-right {
+    display: flex;
+    align-items: center;
+    position: relative;
+    height: 100%;
+    flex: 1;
+  }
+
+  > .iui-center {
+    justify-content: center;
+    @include themed {
+      color: t(iui-text-color);
+    }
+
+    .iui-button-group {
+      margin: 0 $iui-s;
+
+      > .iui-button,
+      > .iui-ellipsis {
+        width: $iui-button-height;
+
+        &.iui-small {
+          width: $iui-button-height-small;
+        }
+      }
+
+      > .iui-ellipsis {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        cursor: default;
+        @include themed {
+          color: t(iui-text-color-muted);
+        }
+      }
+
+      > .iui-active::after {
+        top: auto;
+        bottom: $iui-xxs;
+      }
+    }
+  }
+
+  > .iui-right {
+    justify-content: flex-end;
```

Comment:
Does it give anything?
I dont like that it is shown as 0 height too...

Reply:
Now that I've removed `position: absolute;`, this is required to right align the select menu.

Without `justify-content: flex-end;`:
![Screen Shot 2021-06-04 at 1 03 16 PM](https://user-images.githubusercontent.com/849817/120837856-3c449c00-c535-11eb-9b9e-d5de00451643.png)

With it:
![Screen Shot 2021-06-04 at 1 03 46 PM](https://user-images.githubusercontent.com/849817/120837950-4e263f00-c535-11eb-8c66-71a0f17fd6a1.png)